modules/installing-aws-load-balancer-operator-cli.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/install-aws-load-balancer-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-installing-aws-load-balancer-operator-cli_{context}"]
+= Installing the AWS Load Balancer Operator with the command line
+
+.Prerequisites
+
+* You have logged in to the {product-title} web console as a user with `cluster-admin` permissions.
+
+* Your cluster is configured with AWS as the platform type and cloud provider.
+
+.Procedure
+
+. Create the `aws-load-balancer-operator` namespace:
+
+[source,terminal]
+----
+oc new-project aws-load-balancer-operator
+----
++
+The following configurations will be automatically created:
+
+.Expected `OperatorGroup` object
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+name: aws-load-balancer-operator
+namespace: aws-load-balancer-operator
+spec:
+upgradeStrategy: Default
+----
+
+.Expected `Subscription` object
+
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: aws-load-balancer-operator
+  namespace: aws-load-balancer-operator
+spec:
+  channel: stable-v1.1
+  installPlanApproval: Automatic
+  name: aws-load-balancer-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+----
+. Apply the configurations by running the following command:
++
+[source,terminal]
+Suggested change
+----
+$ oc apply -f 
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+name: aws-load-balancer-operator
+namespace: aws-load-balancer-operator
+spec:
+upgradeStrategy: Default DO WE NEED THE FILENAME HERE
+----
+. Create a YAML file <Subscription> with the following configuration:
++
+[source,yaml]
+----
+CODE HELP NEEDED
+----
+. Apply the configuration by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <name_of_OperatorGroup_config>
+----
++
+[source,terminal]
+----
+$ oc apply -f <name_of_Subscription_config>
```

Comment:
Similar one line command can be used for the creation of `Subscription`:
```
cat <<EOF | oc apply -f -
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: aws-load-balancer-operator
  namespace: aws-load-balancer-operator
spec:
  channel: stable-v1
  name: aws-load-balancer-operator
  source: redhat-operators
  sourceNamespace: openshift-marketplace
EOF
```

Reply:
We like to break up commands into separate steps to be clear about what each step/line of code does, per our style guide.

Reply:
You can do this then:

```bash
cat <<EOF > filename.yaml
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: aws-load-balancer-operator
  namespace: aws-load-balancer-operator
spec:
  channel: stable-v1
  name: aws-load-balancer-operator
  source: redhat-operators
  sourceNamespace: openshift-marketplace
EOF
```

```bash
oc apply -f filename.yaml
```

Reply:
Here is the the yaml, in case it's the way to go:
```yaml
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: aws-load-balancer-operator
  namespace: aws-load-balancer-operator
spec:
  channel: stable-v1
  name: aws-load-balancer-operator
  source: redhat-operators
  sourceNamespace: openshift-marketplace
```